example/program1.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	_ "context"
+	"database/sql"
+	"fmt"
+	_ "log"
+
+	"github.com/IBM/nzgo"
+)
+
+func main() {
+
+	var conninfo string = "host=localhost user=admin password=password dbname=db1 port=5480 sslmode=disable "
+
+	var elog nzgo.PDALogger
+	elog.LogLevel = "debug"
+	elog.LogPath = "/tmp/"
+	elog.Initialize()
```

Comment:
Not required this now. 
We can use configuration parameters and set in conninfo string
logLevel=debug
logPath=/tmp/

same in all examples